client/reader/stream/style.scss

```diff
@@ -1076,23 +1076,6 @@
 	}
 
 	&.search-stream {
-		.back-button {
```

Comment:
This `back-button` is common across multple screens of Reader so moving the styles to `reader/styles.scss` file